src/db/options.ts

```diff
@@ -0,0 +1,23 @@
+import { pgTable, timestamp, integer, uuid, varchar } from 'drizzle-orm/pg-core';
+import { cycles } from './cycles';
+import { relations } from 'drizzle-orm';
+
+export const options = pgTable('options', {
+  id: uuid('id').primaryKey().defaultRandom(),
+  cycleId: uuid('id')
+    .references(() => cycles.id)
+    .notNull(),
```

Comment:
this should be a reference to a question.

the current relationship looks like this
```
cycles 1 - M questions
questions 1 - M options
```